src/components/tables/UsersTable.vue

```diff
@@ -1,95 +1,135 @@
 <template>
-  <q-table
-    class="shadow-5"
-    table-header-class="bg-grey-3"
-    row-key="login"
-    :pagination="pagination"
-    :columns="columns"
-    :rows="users"
-    data-cy="users_table"
-  >
-    <template #body-cell-name="cell">
-      <q-td
-        :props="cell"
-      >
-        <user-avatar
-          :login="cell.row.login"
-          small
-        />
-        <span class="q-pl-md">
-          {{ cell.row.name }}
-        </span>
-      </q-td>
-    </template>
-    <template #body-cell-actions="cell">
-      <q-td
-        key="actions"
-        :props="cell"
-      >
-        <q-btn
-          v-if="showAction"
-          dense
-          flat
-          rounded
-          color="primary"
-          :icon="t('UsersTable.icon.showAction')"
-          :title="t('UsersTable.text.showAction')"
-          :data-cy="`user_${cell.row.login}_button_show`"
-          @click="$emit('show', cell.row.login)"
-        />
-        <q-btn
-          v-if="detachAction"
-          dense
-          flat
-          rounded
-          color="negative"
-          :icon="t('UsersTable.icon.detachAction')"
-          :title="t('UsersTable.text.detachAction')"
-          :data-cy="`user_${cell.row.login}_button_detach`"
-          @click="$emit('detach', cell.row)"
-        />
-        <q-btn
-          v-if="removeAction"
-          dense
-          flat
-          rounded
-          color="negative"
-          :icon="t('UsersTable.icon.removeAction')"
-          :title="t('UsersTable.text.removeAction')"
-          :data-cy="`user_${cell.row.login}_button_remove`"
-          @click="$emit('remove', cell.row)"
-        />
-      </q-td>
-    </template>
-    <template #no-data>
-      <div
-        class="full-width row flex-center q-gutter-sm"
-        data-cy="users_table_no_data"
-      >
-        <q-icon
-          v-if="noDataIcon"
-          size="2em"
-          :name="noDataIcon"
-        />
-        <span v-if="noDataLabel">
-          {{ noDataLabel }}
-        </span>
-      </div>
-    </template>
-  </q-table>
+  <div>
+    <div
+      class="row justify-between items-center q-mb-md"
+    >
+      <user-filters-card
+        :name="filterName"
+        :login="filterLogin"
+        :email="filterEmail"
+        @update:name="setFilterName"
+        @update:login="setFilterLogin"
+        @update:email="setFilterEmail"
+      />
+      <table-pagination-card
+        :current="currentPage"
+        :max="maxPage"
+        :total="totalElements"
+        :size="elementsPerPage"
+        @update:current="setCurrentPage"
+        @update:size="setElementsPerPage"
+      />
+    </div>
+    <q-table
+      v-bind="$attrs"
+      :pagination="pagination"
+      :columns="columns"
+      :rows="users"
+      row-key="login"
+      :loading="loading"
+      class="shadow-5"
+      table-header-class="bg-grey-3"
+      data-cy="users_table"
+    >
+      <template #body-cell-name="cell">
+        <q-td
```

Comment:
```
<q-td :props="cell">
  <user-avatar
    :login="cell.row.login"
    small
  />
</q-td>
<q-td :props="cell">
  <span class="q-pl-md">
    {{ cell.row.name }}
  </span>
</q-td>
```
Be carefull, I'm not sure if it's working. But the purpose is to have one cell for avatar and another one for login.

Reply:
done